digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1674,7 +1674,9 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     logger.info(s"Found ${expiringRoadAddresses.size} to expire; expected ${replacements.map(_.roadAddressId).toSet.size}")
     ProjectDAO.moveProjectLinksToHistory(projectID)
     logger.info(s"Moving project links to project link history.")
-    handleNewRoadNames(projectLinks, project)
+    val validLinksToNames = projectLinks.filter(_.roadNumber <= maxRoadNumberDemandingRoadName)
+    if (validLinksToNames.nonEmpty)
+      handleNewRoadNames(validLinksToNames, project)
```

Comment:
Road number 70000 doesn't need name. Here road number 70001 is the first road which doesn't need name. Also the roads with number 70000 or over CAN be named but it's not mandatory and if they are named we should save the name to the database.